eng/pipelines/templates/steps/test.yml

```diff
@@ -39,6 +43,14 @@ steps:
     displayName: "Test libraries"
     condition: and(succeeded(),eq(variables['TestType'], 'browser'))
 
+  - ${{ if eq(parameters.TestProxy, true) }}:
+    - pwsh: |
+        # ambitious_azsdk_test_proxy is the hardcoded container name used
+        # by the test proxy startup script
+        docker logs ambitious_azsdk_test_proxy
+      displayName: 'Dump Test Proxy logs'
+      condition: succeededOrFailed()
+  
```

Comment:
```suggestion
```
Also removing this @scbedd 

Do you have an alternative for this?

Reply:
I _will_. [issue filed](https://github.com/Azure/azure-sdk-tools/issues/2243)

Reply:
[And PR](https://github.com/Azure/azure-sdk-tools/pull/2252/files)